content/docs/user-guide/external-dependencies.md

```diff
@@ -35,40 +35,44 @@ directory.
 ## Examples
 
 As examples, let's take a look at a [stage](/doc/command-reference/run) that
-simply moves a local file from an external location, producing a `data.txt.dvc`
-stage file (DVC-file).
+simply downloads a file from an external location, adding a `download_file`
+stage to your list of stages in dvc.yaml.
 
 > Note that some of these commands use the `/home/shared` directory, typical in
 > Linux distributions.
 
 ### Local file system path
 
 ```dvc
-$ dvc run -d /home/shared/data.txt \
+$ dvc run -n download_file
+          -d /home/shared/data.txt \
           -o data.txt \
           cp /home/shared/data.txt data.txt
```

Comment:
It's not downloaded in the case of the local file system, where `cp` is used. But OK, the top case covers the base case/ intention. Maybe we should just add a note about this under the **Local file system path** header.

Reply:
Done in 42b670f1.